lib/slack/real_time/client.rb

```diff
@@ -125,6 +127,7 @@ def socket_class
 
       def send_json(data)
         fail ClientNotStartedError unless started?
+        logger.debug("#{self.class.name}#send_data") { data }
```

Comment:
shouldn't this be `#send_json` ? maybe using `"#{self.class}#${__method__}"` would make it automatic.